docs/en/stack/ml/anomaly-detection/ml-ad-job-types.asciidoc

```diff
@@ -0,0 +1,131 @@
+[[ml-anomaly-detection-job-types]]
+= {anomaly-detect-cap} job types
+++++
+<titleabbrev>Job types</titleabbrev>
+++++
+
+:keywords: {ml-init}, {stack}, {anomaly-detect}
+
+:description: The description of the different anomaly detection job types. 
+
+{anomaly-jobs-cap} have many possible configuration options which enable you to 
+fine-tune the jobs and cover your use case as much as possible. This page 
+provides a quick overview of the different types of {anomaly-jobs} and their 
+capabilities. The currently available job types are:
+
+* single metric jobs,
+* multi-metric jobs,
+* population jobs,
+* advanced jobs,
+* categorization jobs,
+* rare jobs.
+
+
+[discrete]
+[[singe-metric-jobs]]
+## Single metric jobs
+
+Every {anomaly-job} has at least one detector. A detector defines the type of 
+analysis that occurs (for example, `max`, `average`, or `high`, it's also called 
```

Comment:
Here "it's" does not apply to the detector but to the occurring analysis (high, max, etc.). I understand that it might be a source of misunderstanding, so I accept the suggestion. Thanks for spotting the issue.